src/dbus/server/introspect.xml

```diff
@@ -339,6 +339,46 @@
     </property>
   </interface>
 
+  <interface name="io.openthread.Commissioner">
```

Comment:
Should this section be in a separate of file because this is describing another object.